.gitignore

```diff
@@ -2,6 +2,8 @@
 .idea
 .vscode
 **/.DS_Store
+scratch_notebook.ipynb
+scratch/
```

Comment:
Create the notebook and then just leave it with a polars import and ignore any data added to the `scratch` folder.